src/Microsoft.TestPlatform.Common/Microsoft.TestPlatform.Common.csproj

```diff
@@ -6,11 +6,12 @@
   <Import Project="$(TestPlatformRoot)scripts/build/TestPlatform.Settings.targets" />
   <PropertyGroup>
     <AssemblyName>Microsoft.VisualStudio.TestPlatform.Common</AssemblyName>
-    <TargetFrameworks>netstandard2.0;netstandard1.3;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard1.3;net451;net6.0</TargetFrameworks>
```

Comment:
Adding net6.0 because I need a TFM greater or equal to netcoreapp3.0 so that we have a dll where `NullableAttributes.cs` is not included. Otherwise we get an error that says the attribute comes from 2 locations (system and this). I am using net6.0 because we are already targeting this for `DotNetBuildFromSource`.

Reply:
Just to confirm. This is not being shipped in any of our Nugets?

Reply:
I have just double checked and running build.cmd locally doesn't produce the net6.0 of the dll in the artifacts folder.